### PR TITLE
Resolve issue #55

### PR DESCRIPTION
**Pull request by abiterry**

### CODE REVIEW

File: R/app.R [hunk 1]
```diff
@@ -1,53 +1,64 @@
...
+    card(
+      h3("Data"),
+      shiny::checkboxGroupInput(
+        inputId = "dataChks",
+        label = "",
+        choices = c("External Data",
+                    "Differing Sample Size per Arm",
+                    "Estimate 'true' Covariate Effects",
+                    "Underlying SD (Normal Case)",
+                    "Robustify Power Prior")
```
Comment (statasaurus) [inline R/app.R:41]:
```suggestion
     
```

File: R/app.R [hunk 2]
```diff
@@ -1,53 +1,64 @@
...
-                                                "Other"))
-                ),
-      nav_panel(title = "Priors", p("Second tab content.")),
-      nav_panel(title = "Posteriors and Inference", p("Third tab content"))
+      ),
+      shiny::radioButtons("covDatGen",
+                          label = "Covariate Data Generation",
+                          choices = c("External Data Based (bootstrap)",
+                                      "Assumed known covariance",
+                                      "Other")),
```
Comment (statasaurus) [inline R/app.R:48]:
```suggestion

```

**Review by statasaurus — CHANGES_REQUESTED** [review]
Looks good. I just have a few suggestions. Also, now that I am thinking about it. It is probably better if we make the binary analysis options a module cause then we can reuse it across the simulation and analysis case

File: R/mod_plot.R [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  }
+  tagList(
+    h4("Plots"),
+    hr(),
+    fluidRow(
+      column(4,
+             shiny::checkboxGroupInput(ns("plotProp"),
+                                       "Propensity Score Plot(s)",
+                                       choices =
+                                         c("Historgram", "Historgram - IPW",
```
Comment (abiterry) [inline R/mod_plot.R:19]: Histogram spelling

Comment (statasaurus) [inline R/mod_plot.R:19]:
```suggestion
                                         c("Histogram", "Histogram - IPW",
```

**Review by statasaurus — APPROVED** [review]

**Pull request merged**